special-calculations/data/constants.js

```diff
@@ -1,5 +1,5 @@
 // constants for use in special calculations
-const INFLATION_FACTOR = 1.1751; // Updated to 2020 value
+const INFLATION_FACTOR = 1.1898; // Updated to 2020 value on 2022-07-20, previously was 1.1751
```

Comment:
Can you just delete this comment? Figuring out when code changed and what it used to be is what Git is for.